modules/titlebar/main/index.coffee

```diff
@@ -23,6 +23,10 @@ setVisibility = (show, animate=true) ->
   animateTitlebar(hx.select(@selector).selectAll('.hx-titlebar-linkbar'))
 
 
+isElement = (obj) -> !!(obj and obj.nodeType is 1)
+
+isSelection = (obj) -> obj instanceof hx.Selection
```

Comment:
@jmsmyth Was going to add these to the util module but wasn't sure if it made sense?

Borrowed `isElement` from [Underscore.js](http://underscorejs.org/#isElement)

Reply:
`isSelection` has also been added in the webpack branch, but in the selection module, so would make sense to add it there if you want to add it to the public api now. Otherwise you could leave it here for now, and when this change gets merged with the webpack branch it can be deduplicated

For `isElement`, there is now a 'dom-utils' module in the webpack branch which would be a good place to put it, or maybe it could also go in the selection module. Probably okay to leave this one here for now

Reply:
Ok, I'll leave them both here and raise a ticket to move them once webpack is merged